components/error/default/index.mjs

```diff
@@ -46,8 +46,14 @@ export class InternalAppmapError extends AppmapError {
 }
 
 export class ExternalAppmapError extends AppmapError {
-  constructor(message) {
+  constructor(message, from = null) {
```

Comment:
Same remark as for your previous dynamic error message. I'm not against improving the current logging mechanism (detailed dynamic log + short static error message) but I don't see the point in changing it in two places where there are like 100+ locations where it is used. In my opinion this should be discussed and changed in a separate PR so that we can group all these change together and retain some consistency.

Reply:
Like I said before, I think short error message needs to contain some specific information to be actually useful. Otherwise it's just annoying and useless. But I agree that it's better to strategically improve all the sites that can benefit from it. I just put these in as I needed to solve the problems I encountered.

OTOH, I don't want this to become another rabbit hole we spend too much time on while we have other issues to deal with. Now I think maybe it's okay to keep those tactical changes and add an issue to address this problem more systematically. If we find the time and impetus to do it while doing some other work, even better, but perhaps we shouldn't take out those tactical changes just for the sake of purity; if I encountered these, perhaps the users will too.